tests/contracts/manager_test.py

```diff
@@ -57,7 +57,7 @@ def test_create_schain_data_to_bytes(skale):
 def test_get_bounty(skale, wallet):
     node_id = skale.nodes_data.node_name_to_index(DEFAULT_NODE_NAME)
     res = skale.manager.get_bounty(node_id, wallet)
-    receipt = Helper.await_receipt(skale.web3, res['tx'])
+    _ = Helper.await_receipt(skale.web3, res['tx'])
```

Comment:
Why **receipt** was renamed to **_**?

Reply:
@badrogger let's add `check_receipt` here